ansible/roles/satellite-capsule-installation/tasks/firewalld.yml

```diff
@@ -0,0 +1,40 @@
+---
+
+- name: firewall | Install Firewalld
+  yum:
+    name: firewalld
+    state: present
+  tags:
+    - configure_capsule_firewalld
+ 
+- name: firewall | Enable and Start Firewalld
+  service:
+    name: firewalld
+    enabled: yes
+    state: started
+  tags:
+    - configure_capsule_firewalld
+
+
```

Comment:
small nit: get rid of extra lines for consistency

Reply:
done